packages/vanilla/src/components/drawer/drawer.stories.js

```diff
@@ -36,56 +36,9 @@ const Template = ({drawerPosition}) => {
       </div>
 
       <nav class="cbp-drawer__content">
```

Comment:
technically you don't need the `nav` tag. If there are button instead of links inside (e.g., filtering), then it's not really nav at all. So if we're going to gut the example, let's make it as dumb as possible.

Reply:
switched with a `<div>` tag